@here/olp-sdk-core/lib/utils/DataStoreDownloadManager.ts

```diff
@@ -131,6 +131,16 @@ export class DataStoreDownloadManager implements DownloadManager {
         );
     }
 
+    private static requestShouldRetryOnError(code: number): boolean {
+        const SERVER_ERROR_CODES_FROM = 500;
+        const SERVER_ERROR_CODES_TO = 599;
```

Comment:
Maybe add these to the STATUS_CODES enum?

Reply:
added